b2share/modules/records/tasks.py

```diff
@@ -60,6 +60,7 @@ def update_expired_embargos():
             ),
             allow_leading_wildcard=False
             ).fields([])
+        
```

Comment:
Can you remove this line? We now try to have as small commits as possible. If style needs to be updated it can be done in a separate commit.